src/api/c++/z3++.h

```diff
@@ -1259,7 +1332,7 @@ namespace z3 {
         friend expr bvneg_no_overflow(expr const& a);
         friend expr bvmul_no_overflow(expr const& a, expr const& b, bool is_signed);
         friend expr bvmul_no_underflow(expr const& a, expr const& b);
-        
```

Comment:
these changes just annoy the reviewer. I've spotted the review here. Too long.

Reply:
Yeah, I'll make sure to turn off my trailing whitespace zapper next time; sorry about that.